addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -725,14 +725,18 @@ blockquote {
     }
 }
 %o-we-background-layer {
-    @include o-position-absolute(0, 0, 0, 0);
+    position: absolute !important;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
```

Comment:
I merged a similar fix recently while fixing another bug so I guess you can just drop this part sorry